tests/unit/consensus/test_decisions_accepted.py

```
@@ -0,0 +1,471 @@
+
+
+# ═══════════════════════════════════════════════════════════════════
+# decide_accepted
+# ═══════════════════════════════════════════════════════════════════
+
+
+class TestDecideAcceptedNormal:
+    """Normal acceptance path: not appealed, not appeal_undetermined."""
+
+    def test_consensus_round_is_accepted(self):
+        pre, post, cr, rv = decide_accepted(**_base_accepted_kwargs())
```

Ruff (0.15.2)

[warning] Unpacked variable `pre` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)